.github/workflows/ci.yml

```diff
@@ -17,17 +17,26 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Set up Go
-        uses: actions/setup-go@v3
+        id: setup-go
+        uses: actions/setup-go@v4
         with:
           go-version-file: go.mod
-          cache: true
+          check-latest: true
 
       - name: Set up Chrome
         uses: browser-actions/setup-chrome@latest
 
       - name: Run lint
         uses: reviewdog/action-golangci-lint@v2
 
+      - name: Run govulncheck
+        uses: golang/govulncheck-action@v1
+        with:
+           go-version-input: '${{ steps.setup-go.outputs.go-version }}'
+           check-latest: true
+           cache: true
```

Comment:
https://github.com/k1LoW/runn/pull/554/files#r1264664860

```suggestion
```

Perhaps removing the use of the cache might work.

Reply:
https://github.com/k1LoW/runn/actions/runs/5567405027/jobs/10169253581?pr=554#step:7:32289

I feel like it's a different issue, so let's Merge once.